vtex-apps/search-result
# Fix - README typo and table formatting

#### What is the purpose of this pull request?
- Fix README typos and table formatting;
- Delete README from root.

#### How should this be manually tested?
- Check if the fixes make sense;
- Check if the README still shows up in the repository home page.

#### Screenshots or example usage
N/A.

#### Types of changes
* Documentation fix.
